aml-development/ozp-backend
# feat(recommend): add api recommendation debug dictionary

AMLNG-921

**Description**     
Add Debug Information to recommend endpoint to allow better debugging

**Acceptance Criteria**   
Debug Info was added to /api/storefront/recommend/?debug=true
 
**How to test code**    
```
git fetch
git checkout recommend_20180123
make dev run
```
Call `http://127.0.0.1:8001/api/storefront/recommended/?debug_recommendations=True`
you should see `recommended_debug` key under results dictionary

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
